models/med_classifer_svm.py

```diff
@@ -0,0 +1,108 @@
+"""
```

Comment:
Similarly, if we're no longer using this file, it may be worthwhile to remove it. Otherwise, we should transform this into a well-presented Jupyter notebook that creates a model file.